modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectDefinitionUtil.java

```diff
@@ -161,6 +161,9 @@ public static ObjectDefinition toObjectDefinition(
 					serviceBuilderObjectDefinition::isEnableIndexSearch);
 				setEnableLocalization(
 					serviceBuilderObjectDefinition::isEnableLocalization);
+				setEnableObjectDefinitionMapping(
+					serviceBuilderObjectDefinition::
+						isEnableObjectDefinitionMapping);
```

Comment:
you should cover this new property with FF :
```
setEnableObjectDefinitionMapping(
	() -> {
		if (!FeatureFlagManagerUtil.isEnabled("LPD-xxxx")) {
			return null;
		}

		return serviceBuilderObjectDefinition.isEnableObjectDefinitionMapping();
	});
```

remember to user the `companyId` in the FF method. Review all usages in your work.